src/cool_seq_tool/mappers/exon_genomic_coords.py

```diff
@@ -1,32 +1,22 @@
 """Provide mapping capabilities between transcript exon and genomic coordinates."""
 
 import logging
-from typing import TypeVar
 
-from pydantic import Field, StrictInt
+from ga4gh.vrs import models
```

Comment:
can we just import SequenceLocation since (I think) that's all we're using?

Reply:
Sure